semgrep/tests/e2e/test_rule_parser.py

```diff
@@ -43,3 +43,16 @@ def test_rule_parser__failure__error_messages(run_semgrep_in_tmp, snapshot, file
 
     if excinfo_in_color.value.stderr != excinfo.value.stderr:
         snapshot.assert_match(excinfo_in_color.value.stderr, "error-in-color.txt")
+
+
+# https://github.com/returntocorp/semgrep/issues/1095
+def test_rule_parser_cli_pattern(run_semgrep_in_tmp, snapshot):
+    # This test should eventually pass once this pattern is handled by semgrep-core, but for now,
+    # getting a proper error from semgrep-core is an improvement
+    with pytest.raises(CalledProcessError) as excinfo:
+        run_semgrep_in_tmp(options=["-e", "#include $X", "-l", "c"])
```

Comment:
Should we just use a different pattern that is actually bad instead of one that might eventually work?